services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/ProtectedPathSpec.groovy

```diff
@@ -253,9 +253,9 @@ class ProtectedPathSpec extends HealthCheckSpecification {
     @Unroll
     def "System reroutes #flowDescription flow to more preferable path and ignores protected path when reroute\
  is intentional"() {
-        given: "Two active neighboring switches with two diverse paths at least"
+        given: "Two active neighboring switches with four diverse paths at least"
         def switchPair = topologyHelper.getAllNeighboringSwitchPairs().find {
-            it.paths.unique(false) { a, b -> a.intersect(b) == [] ? 1 : 0 }.size() >= 2
+            it.paths.unique(false) { a, b -> a.intersect(b) == [] ? 1 : 0 }.size() >= 4 //issue-2762
```

Comment:
I don't think this is possible neither at our virtual nor at staging topology, thus it is doomed to always be skipped now with "No suiting switches found"

Reply:
I don't really understand what means `//issue-2762`. Is picking a path reproduces this issue? no sense

Reply:
Nobody sees a 'diff' when looking at this code. After this pr it would have became
`it.paths.unique(false) { a, b -> a.intersect(b) == [] ? 1 : 0 }.size() >= 4 //issue-2762`